CRM/Core/Config/MagicMerge.php

```diff
@@ -232,9 +232,11 @@ public function __get($k) {
 
       case 'setting-url-abs':
       case 'setting-url-rel':
+      case 'setting-url-rel-noslash':
+        $noslash = ($type == 'setting-url-abs-noslash') ? TRUE : FALSE;
         $type = (strstr($type, 'abs')) ? 'absolute' : 'relative';
         $value = $this->getSettings()->get($name);
-        if ($value) {
+        if ($value && !$noslash) {
           $value = CRM_Utils_File::addTrailingSlash($value);
```

Comment:
@totten If this case really is dealing with urls then we need to pass '/' as the 2nd param to addTrailingSlash.